multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/OperationsHelper.java

```diff
@@ -67,52 +56,4 @@ private ErrorType toErrorType(HistoricOperationEvent.EventType historicType) {
         }
         return null;
     }
-
-    public Operation addState(Operation operation) {
-        if (operation.getState() != null) {
-            return operation;
-        }
-        Operation.State state = computeState(operation);
-        // Fixes bug XSBUG-2035: Inconsistency in 'operation', 'act_hi_procinst' and 'act_ru_execution' tables
```

Comment:
was this code used to release a lock on every operation state poll? will the mentioned inconsistency come up again with it deleted?